dask/dataframe/io/tests/test_parquet.py

```diff
@@ -4490,3 +4490,27 @@ def test_select_filtered_column(tmp_path, engine):
     with pytest.warns(UserWarning, match="Sorted columns detected"):
         ddf = dd.read_parquet(path, engine=engine, filters=[("b", "==", "cat")])
     assert_eq(df, ddf)
+
+
+@PYARROW_MARK
+@pytest.mark.parametrize("pass_schema", [True, False])
```

Comment:
@rjzamora Instead of the `pass_schema` param, perhaps parametrize with possible schema options, to also test the dict option:

```python
"schema", [
  None,
  {"a": pa.string()},
  pa.schema([pa.field("a", pa.string())]),
]
```

Reply:
Good call!